extensions/vscode/webview/src/main.ts

```
@@ -56,6 +59,76 @@ export function base64_to_array_buffer(base64: string): ArrayBuffer {
   return bytes.buffer
 }
 
+// Apply theme to the webview DOM
+const apply_theme = (theme: ThemeName): void => {
+  const root = document.documentElement
+
+  // Set the data-theme attribute
+  root.setAttribute(`data-theme`, theme)
+
+  // Apply MatterViz theme if available
+  if (typeof globalThis !== `undefined` && globalThis.MATTERVIZ_THEMES) {
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)